tests/CoffeeNation.UnitTestsCommon/MockData.cs

```diff
@@ -0,0 +1,197 @@
+        };
+
+
+
+        // Distance
+        public static Distance NullShopDistance => null;
+
+        public static Distance ShopDistance1 => new Distance
+        {
+            Tag = "Coffee Shop1",
+            Value = 423.21343245
+        };
+
+        public static Distance ShopDistance2 => new Distance
+        {
+            Tag = "Coffee-Shop2",
+            Value = 56.234568
+        };
+
+        public static Distance ShopDistance3 => new Distance
+        {
+            Tag = "CoffeeShop3",
+            Value = 24.5248765
+        };
+
+        public static Distance ShopDistance4 => new Distance
+        {
+            Tag = "Coffee Shop4",
+            Value = 5.325
+        };
+
+        public static Distance ShopDistance5 => new Distance
+        {
+            Tag = "Coffee Shop5",
+            Value = 7.86535
+        };
+
+        public static Distance ShopDistance99 => new Distance
+        {
+            Tag = "Starbucks Seattle2",
+            Value = 0.0645
+        };
+
+        public static IEnumerable<Distance> NullShopDistances => null;
+
+        public static IEnumerable<Distance> EmptyShopDistances => new List<Distance>();
+
+        public static IEnumerable<Distance> LessThanThreeShopDistances => new List<Distance>
+        {
+            ShopDistance1,
+            ShopDistance2
+        };
+
+        public static IEnumerable<Distance> AllShopDistances => new List<Distance>
+        {
+            ShopDistance1,
+            ShopDistance2,
+            ShopDistance3,
+            ShopDistance4,
+            ShopDistance5
+        };
+
+        public static IEnumerable<Distance> SelectedShopDistances => new List<Distance>
+        {
+            ShopDistance4,
+            ShopDistance5,
+            ShopDistance3
+        };
+
+        public const string FormattedCoffeeShopDistance1 = "Coffee Shop1,423.2134";
+
+        // Csv content
+        public static string StringNullCsvLine => null;
+        public static string StringEmptyCsvLine => null;
+        public static string LessThanThreeTokensCsvLine => "Starbucks Seattle,47.5809";
+        public static string MoreThanThreeTokensCsvLine => "Starbucks Seattle,47.5809,-122.3160,asd";
+        public static string Token1ErrorCsvLine => ",47.5809,-122.3160";
+        public static string Token2ErrorCsvLine => "Starbucks Seattle,asd,-122.3160";
+        public static string Token3ErrorCsvLine => "Starbucks Seattle,47.5809,qwe";
+
+        public static string ValidCsvLine1 => "Starbucks Seattle,47.5809,-122.3160";
+        public static string ValidCsvLine2 => "Starbucks SF,37.5209,-122.3340";
+        public static string ValidCsvLine3 => "Starbucks Moscow,55.752047,37.595242";
+
+        public static string ValidCsvLine99 => "Starbucks Seattle,47.5809,-122.3160";
+
+        public static IEnumerable<string> ValidCsvContent => new List<string>
+        {
+            ValidCsvLine1,
+            ValidCsvLine2,
+            ValidCsvLine3
+        };
```

Comment:
Agreed, the file will grow as the app grows and we can split it into partial classes when we need to, or even different classes if we feel the need to. Its purpose is to have all the mocked data into one place so that is easier to model the context for tests, in case objects that are somehow dependent need to be passed. It should help with giving meaning to the test cases not having magic values.